arazzo/museum-api.arazzo.yaml

```diff
@@ -4,12 +4,13 @@ info:
   description: >-
     Use the Museum API with Arazzo as an example of describing multi-step workflows.
     Built with love by Redocly.
-  version: 1.0.0
+  version: 1.0.1
 
 sourceDescriptions:
   - name: museum-api
     type: openapi
     url: ../openapi.yaml
+    x-serverUrl: "http://localhost:4000/_mock/openapi" # For local development testing
```

Comment:
Last time I added changes to this file I  intentionally updated the `openapi.yaml` with mock-server
```
servers:
  - url: "https://redocly.com/_mock/docs/openapi/museum-api"
```
  
  The main idea was that users don't need to set any mock-server on their one and Spot run will work out of the box.